extensions/arc/deployment/src/main/resources/dev-templates/interceptors.html

```diff
@@ -0,0 +1,77 @@
+{#include main fluid=true}
+  {#style}
+   .annotation {
+     color: gray;
+     font-style: italic;
+   }
+   span.larger-badge {
+     font-size: 0.9em;
+   }
+   span.app-class {
+     cursor:pointer;
+     color:blue;
+     text-decoration:underline;
+   }
+
+  {/style}
+
+  {#script}
+    $(document).ready(function(){
+      if (!ideKnown()) {
+        return;
+      }
+      $(".class-candidate").each(function() {
+        var className = $(this).text();
+        if (appClassLang(className)) {
+          $(this).addClass("app-class");
+        }
+      });
+
+      $(".app-class").on("click", function() {
+        openInIDE($(this).text());
+      });
+    });
+
+  {/script}
+
+  {#title}Interceptors{/title}
+  {#body}
+  <div class="alert alert-primary alert-dismissible fade show" role="alert" data-timer="30000">
+    Interceptors are sorted by the interceptor class in ascending order. However, application interceptors go first. 
```

Comment:
Wouldn't it make sense to sort by priorioty?

Reply:
Agree that this would make a lot of sense, but I think that showing application interceptors more prominently than platform/external ones also makes sense.

Reply:
Maybe they should instead have a different color or some marker?

I am definitely no UI expert, so I have no idea what the best representation should be...

Reply:
So right now we use `isApplicationClass` and then `interceptorClass` name. We could probably change this to `isApplicationClass` and then `priority` name and then `interceptorClass` name...

Reply:
> Maybe they should instead have a different color or some marker?

They do - application classes are links that can be opened in your IDE... except it does not work for me ;-).

Reply:
I have never been able to test Eclipse for this. A little while ago, @FroMage did and it worked for him